docs/application/dotnet/guides/nui/view.md

```diff
@@ -133,4 +135,3 @@ private bool ViewTouchEventCallBack(object sender, View.TouchEventArgs e)
 ## Related Information
```

Comment:
@MathewP1 Request you to change the headings style to sentence case? 
Please refer to: 
https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization

Is it fine for you? 
So, 

 ## Related Information 

becomes 

 ## Related information

Reply:
I've changed heading style according to your suggestion

Reply:
@mathewP1 thanks